db/db_impl.cc

```diff
@@ -3066,7 +3064,7 @@ Status DBImpl::VerifyChecksum() {
         const auto& fd = vstorage->LevelFilesBrief(i).files[j].fd;
         std::string fname = TableFileName(cfd->ioptions()->cf_paths,
                                           fd.GetNumber(), fd.GetPathId());
-        s = rocksdb::VerifySstFileChecksum(options, env_options, fname);
+        s = rocksdb::VerifySstFileChecksum(GetOptions(), env_options_, fname);
```

Comment:
`GetOptions()` returns the options built from the default column family. We are iterating different column families here, does this mean we need to call `GetOptions` on each of them?

Reply:
Yeah i think you're right! wasn't sure of the best way to get the options per CF and didn't really think about it, let me fix it

Reply:
From `DBImpl::GetOptions`'s implementation, I think `Options(BuildDBOptions(immutable_db_options_, mutable_db_options_), cfd->GetLatestCFOptions())` should work.